bin/configs/java-resttemplate-list-schema-validation.yaml

```diff
@@ -0,0 +1,11 @@
+generatorName: java
+outputDir: samples/client/petstore/java/resttemplate-list-schema-validation
```

Comment:
can you please add the output folder to `.github/workflows/samples-java-client-jdk11.yaml`?